apps/et/et-sya/demo-image-policy.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     hmcts.github.com/prod-automated: disabled
 spec:
   filterTags:
-    pattern: '^prod-[a-f0-9]+-(?P<ts>[0-9]+)'
```

Comment:
You might need to change {"$imagepolicy": "flux-system:et-sya"} to {"$imagepolicy": "flux-system:demo-et-sya"} here as well:
https://github.com/hmcts/cnp-flux-config/blob/master/apps/et/et-sya/demo.yaml#L10